lib/galaxy/tools/parameters/basic.py

```diff
@@ -38,7 +37,12 @@
 
 log = logging.getLogger(__name__)
 
-workflow_building_modes = Bunch(DISABLED=False, ENABLED=True, USE_HISTORY=1)
+
+class workflow_building_modes:
```

Comment:
Should this be an enum ?

Reply:
We don't iterate over the keys or use them as anything but constants and the types don't match. I guess we could get slightly better typing if we got this to work as an enum? I'm not sure.